### PR TITLE
[v1.0.x] *: fix run packagemanifests

### PR DESCRIPTION
**Pull request by joelanford**
**Description of the change:**
Backport #3856 and #3867 

**Motivation for the change:**
Supported branches should get bug fixes. This unblocks CI, which is currently failing due to issues with OLM v0.16.1's caching of catalog data.

**Checklist**

If the pull request includes user-facing changes, extra documentation is required:
- [ ] Add a new changelog fragment in `changelog/fragments` (see [`changelog/fragments/00-template.yaml`](https://github.com/operator-framework/operator-sdk/tree/master/changelog/fragments/00-template.yaml))
- [ ] Add or update relevant sections of the docs website in [`website/content/en/docs`](https://github.com/operator-framework/operator-sdk/tree/master/website/content/en/docs)

### CODE REVIEW

File: test/e2e-helm/e2e_helm_olm_test.go [hunk 1]
```diff
@@ -44,16 +43,18 @@ var _ = Describe("Integrating Helm Projects with OLM", func() {
...
 			err = tc.Make("bundle-build", "BUNDLE_IMG="+bundleImage)
 			Expect(err).NotTo(HaveOccurred())
 
-			By("loading the project image into Kind cluster")
-			err = tc.LoadImageToKindClusterWithName(bundleImage)
-			Expect(err).Should(Succeed())
+			if isRunningOnKind() {
+				By("loading the bundle image into Kind cluster")
+				err = tc.LoadImageToKindClusterWithName(bundleImage)
+				Expect(err).Should(Succeed())
```
Comment (hasbro17) [inline test/e2e-helm/e2e_helm_olm_test.go:56]: Nit: I know this is a backport and this line is basically unchanged in this PR but for better readability this probably should be:
```Go
Expect(err).NotTo(HaveOccurred())
```
`Succeed()` is used more for the function itself:
```Go
Ω(DoSomethingSimple()).Should(Succeed())
```
https://onsi.github.io/gomega/#handling-errors

**Review by hasbro17 — APPROVED** [review]
LGTM

File: test/e2e-helm/e2e_helm_olm_test.go (hunk 1 above)
Comment (joelanford) [reply test/e2e-helm/e2e_helm_olm_test.go:56]: +1 I'll make this comment into an issue. There are probably lots of instances of this that we can fix on master (probably not worth backporting those changes though).

**Pull request merged**